mmv1/products/apigee/EnvKeystore.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+  name: 'EnvKeystore'
```

Comment:
We typically use `google_product_resource` as the resource name, rather than `google_product_parent(s)_resource`, so this would just be `Keystore` based on the type name in https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.environments.keystores#resource:-keystore. I don't see a reason based on the current API to deviate here!

(same for other resources)